test/service_instance_test/service_instance_test.go

```diff
@@ -2668,3 +2899,11 @@ func prepareBrokerWithCatalog(ctx *TestContext, auth *SMExpect) (*BrokerUtils, *
 
 	return brokerUtils, auth.ListWithQuery(web.ServicePlansURL, "fieldQuery="+fmt.Sprintf("service_offering_id eq '%s'", so.Object().Value("id").String().Raw()))
 }
+
+func findPlanIDForCatalogID(ctx *TestContext, brokerID, catalogID string) string {
+	soID := ctx.SMWithOAuth.ListWithQuery(web.ServiceOfferingsURL, fmt.Sprintf("fieldQuery=broker_id eq '%s'", brokerID)).
+		First().Object().Value("id").String().Raw()
```

Comment:
will not work if the the broker has multiple offerings and the plan is not in the first one